packages/devtools_app/lib/src/shared/globals.dart

```diff
@@ -87,3 +88,12 @@ void setStagerMode() {
     _stagerMode = true;
   }
 }
+
+/// Current mode of DevTools.
+DevToolsMode get devToolsMode {
+  return offlineDataController.showingOfflineData.value
+      ? DevToolsMode.offlineData
+      : serviceConnection.serviceManager.hasConnection
+          ? DevToolsMode.connected
+          : DevToolsMode.disconnected;
+}
```

Comment:
where will this be used? This seems like it is more reflective of the DevTools "screen" mode than it is the global state of DevTools. Can we move this to the screen class if that is where this getter is going to be used?

Reply:
It will be used in controller's constructor.

Reply:
May be it is better to put in its own lib?

Reply:
Maybe put this in `utils.dart` instead of globals

Reply:
done